src/main/java/org/broadinstitute/hellbender/tools/funcotator/mafOutput/MafOutputRenderer.java

```diff
@@ -482,6 +493,106 @@ else if ( value.toLowerCase().startsWith("chr") ) {
+                                MafOutputRendererConstants.VariantClassificationMap.get(GencodeFuncotation.VariantClassification.START_CODON_DEL.toString()),
+                                MafOutputRendererConstants.VariantClassificationMap.get(GencodeFuncotation.VariantClassification.NONSTOP.toString()),
+                                MafOutputRendererConstants.VariantClassificationMap.get(GencodeFuncotation.VariantClassification.FIVE_PRIME_UTR.toString()),
+                                MafOutputRendererConstants.VariantClassificationMap.get(GencodeFuncotation.VariantClassification.THREE_PRIME_UTR.toString()),
+                                MafOutputRendererConstants.VariantClassificationMap.get(GencodeFuncotation.VariantClassification.FIVE_PRIME_FLANK.toString()),
+                                MafOutputRendererConstants.VariantClassificationMap.get(GencodeFuncotation.VariantClassification.INTRON.toString())
+                        },
+                        new String[]{
+                                GencodeFuncotation.VariantClassification.IN_FRAME_DEL.toString(),
+                                GencodeFuncotation.VariantClassification.IN_FRAME_INS.toString(),
+                                GencodeFuncotation.VariantClassification.FRAME_SHIFT_INS.toString(),
+                                GencodeFuncotation.VariantClassification.FRAME_SHIFT_DEL.toString(),
+                                GencodeFuncotation.VariantClassification.MISSENSE.toString(),
+                                GencodeFuncotation.VariantClassification.NONSENSE.toString(),
+                                GencodeFuncotation.VariantClassification.SILENT.toString(),
+                                GencodeFuncotation.VariantClassification.SPLICE_SITE.toString(),
+                                GencodeFuncotation.VariantClassification.DE_NOVO_START_IN_FRAME.toString(),
+                                GencodeFuncotation.VariantClassification.DE_NOVO_START_OUT_FRAME.toString(),
+                                GencodeFuncotation.VariantClassification.START_CODON_SNP.toString(),
+                                GencodeFuncotation.VariantClassification.START_CODON_INS.toString(),
+                                GencodeFuncotation.VariantClassification.START_CODON_DEL.toString(),
+                                GencodeFuncotation.VariantClassification.NONSTOP.toString(),
+                                GencodeFuncotation.VariantClassification.FIVE_PRIME_UTR.toString(),
+                                GencodeFuncotation.VariantClassification.THREE_PRIME_UTR.toString(),
+                                GencodeFuncotation.VariantClassification.FIVE_PRIME_FLANK.toString(),
+                                GencodeFuncotation.VariantClassification.INTRON.toString(),
+                        }
+                );
+
+                // Handle the RNA/LINCRNA case specially:
```

Comment:
Why does this need to be addressed in a special case?

Reply:
Because the replacement string contains the needle string (i.e. `LINCRNA` contains `RNA`).  This causes the replace repeatedly method to throw an exception after replacing the `RNA` in `LINCRNA` with `LINCRNA` many times.